console/views/store.py

```diff
@@ -31,7 +31,7 @@ def get_filtered_responses(logger, valid, tx_id, ru_ref, survey_id, datetime_ear
         if valid == "invalid":
             q = q.filter(SurveyResponse.invalid)
         elif valid == "valid":
-            q = q.filter(not SurveyResponse.invalid)
+            q = q.filter(SurveyResponse.invalid == False)
```

Comment:
This is fine for sqlalchemy but not python, easiest way would be to add `# noqa`